cmd/privateendpoints-setuplink.go

```diff
@@ -22,7 +24,8 @@ var privateEndpointsSetupLinkCmd = &cobra.Command{
 		shouldAutoConfig, _ := cmd.Flags().GetBool("auto")
 		instanceId, _ := cmd.Flags().GetString("instance-id")
 		vmId, _ := cmd.Flags().GetString("vm-id")
-
+		instanceIdGcp, _ := cmd.Flags().GetString("gcp-instance-id")
+		projectIdGcp, _ := cmd.Flags().GetString("gcp-project-id")
```

Comment:
I think this should be something that can be automatically determined based on the instance-id?  Or perhaps the project-id is actually something that should be defined in the configuration or as part of your GCP CLI configuration?

Reply:
Yes, I recently planned to remove this from setup-link command and add it to GCP CLI configuration.